openfisca_france/model/prelevements_obligatoires/impot_revenu/prelevements_forfaitaires/ir_prelevement_forfaitaire_unique.py

```diff
@@ -92,11 +92,12 @@ class assurance_vie_pfu_ir(Variable):
     definition_period = YEAR
 
     def formula_2018_01_01(foyer_fiscal, period):
+        choix_bareme = foyer_fiscal('f2op', period)
         f2zz = foyer_fiscal('f2zz', period)
         f2vv = foyer_fiscal('f2vv', period)
         f2ww = foyer_fiscal('f2ww', period)
 
-        return f2zz + f2vv + f2ww
+        return (choix_bareme == 0) * (f2zz + f2vv + f2ww)
```

Comment:
La variable `choix_bareme` est un booléen donc pas besoin de `== 0`.
Et d'ailleurs, il est possible d'utiliser `where` qui est particulièrement adaptée [aux choix binaires](https://openfisca.org/doc/coding-the-legislation/25_vectorial_computing.html#ternaries).
Et je trouve que le nom de variable `choix_bareme` n'est pas très clair, je suggérerai `imposition_au_bareme`.

Reply:
Je suis tout à fait d'accord pour le nom de la variable, je viens de faire un nouveau commit dans ce sens. J'ai également modifié les `== 0` en `not` pour que ce soit plus lisible, dis moi si cela vous convient ou si tu préfères une utilisation de `where`.